### PR TITLE
Operator bundle support

### PR DESCRIPTION
**Pull request by MartinBasti**
Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

TODO:
- [x] changes in koji_import (at least new subtype `operator_bundle`)
~- [ ] mutual exclusivity appregistry vs bundle~ (will be separated PR)

### CODE REVIEW

**Comment by MartinBasti** [comment]
Depends on https://github.com/containerbuildsystem/osbs-client/pull/939

**Comment by MartinBasti** [comment]
>  mutual exclusivity appregistry vs bundle

I'm not sure where to put this check, it should happen early because we don;t want to pull everything and then fail. I feel this may be a new plugin right after reading reactor config. Or maybe this can be part of `pre_pin_digest_plugin`

File: tests/test_util.py [hunk 1]
```diff
@@ -1642,3 +1645,49 @@ def test_allow_repo_dir_in_dockerignore(tmpdir, dockerignore_exists):
...
+        df_content += 'LABEL {}\n'.format(label)
+    workflow.source = StubSource()
+    workflow.builder = StubInsideBuilder()
+    cont_path = os.path.join(str(tmpdir), 'Dockerfile')
+    with open(cont_path, 'w') as f:
+        f.write(df_content)
+    workflow.builder.df_path = cont_path
+
+    for func in f_true:
+        assert func(workflow), 'Label not properly detect'
```
Comment (chmeliik) [inline tests/test_util.py]: Typo detect[ed]

**Comment by MartinBasti** [comment]
Release notes:
> - Support for operator bundles defined with `com.redhat.delivery.operator.bundle` label has been added

**Pull request merged**